test/new-e2e/tests/windows/components/defender/component.go

```diff
@@ -33,7 +33,7 @@ func NewDefender(e *config.CommonEnvironment, host *remote.Host, options ...Opti
 	}
 
 	manager := &Manager{
-		namer: e.CommonNamer.WithPrefix("windows-defender"),
+		namer: e.CommonNamer().WithPrefix("windows-defender"),
```

Comment:
This change doesn't seem relevant to centos AMIs.
it also does not occur in the referenced PR #27197 
Is this intentional?

Reply:
Hi, I had to bump `test-infra-definitions` as I needed an image compatible with the version that updates the AMI, and we release `test-infra` images only on `main` currently. This version contains some APIs changes, that I applied. Changes should be minimal and trivial to review. Here for instance we move from a property to a method, same name.

I updated the PR description with some context

Reply:
The changes do not occur in https://github.com/DataDog/datadog-agent/pull/27197 because they were already applied.